src/OlStyleParser.ts

```diff
@@ -610,23 +610,24 @@ export class OlStyleParser implements StyleParser<OlStyleLike> {
       // ScaleDenominator and Filters are completly supported so we just check for symbolizers
       rule.symbolizers.forEach(symbolizer => {
         const key = capitalizeFirstLetter(`${symbolizer.kind}Symbolizer`);
-        const value = this.unsupportedProperties?.Symbolizer?.[key];
+        const value = this.unsupportedProperties?.Symbolizer?.[key as keyof UnsupportedProperties['Symbolizer']];
```

Comment:
Can you extract `key as keyof UnsupportedProperties['Symbolizer']`  this to a new `type`?
Would enhance readability and ... Same for the other appearances of `key as key of ...`

Reply:
Done in 4fac030